Mathlib/Topology/Separation.lean

```diff
@@ -1922,29 +1918,29 @@ end CompletelyNormal
 
 /-- In a compact t2 space, the connected component of a point equals the intersection of all
 its clopen neighbourhoods. -/
-theorem connectedComponent_eq_iInter_clopen [T2Space α] [CompactSpace α] (x : α) :
-    connectedComponent x = ⋂ Z : { Z : Set α // IsClopen Z ∧ x ∈ Z }, Z := by
+theorem connectedComponent_eq_iInter_clopen [T2Space X] [CompactSpace X] (x : X) :
+    connectedComponent x = ⋂ Z : { Z : Set X // IsClopen Z ∧ x ∈ Z }, Z := by
```

Comment:
Here things are perhaps worse because `X` looks like `Z` but one is the topogical space and the other is a subtype of sets.

Reply:
I agree this is not the best notation. (I've seen such in mathematical textbooks, though.)
Better names welcome. `s` instead of `Z`?

Reply:
Pushed a commit using `s` instead.